modules/core/server/services/influx.server.service.js

```diff
@@ -44,12 +46,12 @@ exports.getClient = function(callback) {
  * influxdb (camelCase)
  * @param {Object} fields - key: value pairs will be saved in influxdb as field
  * key: field value
- * @param {number|Date} [fields.time=new Date()] - time of measurement with precision ms
+ * @param {number|Date} [fields.time=new Date()] - time of measurement with precision nanosecond
```

Comment:
I updated this based on this comment from changelog: "_write methods now write in nanosecond precision by default_". Any idea if I should change any code regarding this?